.checkstyle/suppressions.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+
+<!DOCTYPE suppressions PUBLIC
+        "-//Puppy Crawl//DTD Suppressions 1.1//EN"
+        "http://www.puppycrawl.com/dtds/suppressions_1_1.dtd">
+
+<suppressions>
+
+    <!-- Note that [/\\] must be used as the path separator for cross-platform support -->
+
+    <suppress checks="NPathComplexity|CyclomaticComplexity"
+              files="io[/\\]strimzi[/\\]kafka[/\\]bridge[/\\]amqp[/\\]converter[/\\]AmqpJsonMessageConverter.java"/>
```

Comment:
Can we use `@SuppressWarnings("checkstyle:NPathComplexity", "checkstyle:CyclomaticComplexity")` on the relevent method instead?